documentation.html

```diff
@@ -392,12 +409,56 @@ <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
               </div>
             </div>
           </section>
-          
+
+        </section>
+
+
+
+        <!-- Dark button class starts here -->
+        <div>
+          <section class="instruction-block" id="dark-mode">
+            <h2 id="base-icon-btn">Dark-Mode Button</h2>
```

Comment:
remove the id as it is not necessary here.

Reply:
I meant id for h2 not for the section. Keep the id of the section for the menu link